components/tools/OmeroJava/test/integration/gateway/TablesFacilityTest.java

```diff
@@ -137,7 +141,7 @@ public void testSearch() throws Exception {
             }
     }
 
-    @Test(dependsOnMethods = { "testAddTable" })
+    @Test(dependsOnMethods = { "testSearch" })
```

Comment:
This needs to be reverted. Running individual tests, shows correct dependencies.